src/lib/mappers/utils.js

```diff
@@ -315,66 +315,6 @@ module.exports = {
                     "@id": "https://www.wikidata.org/wiki/Q1809071",
                     "label": "Design Museum Gent"
                 };
-            } else if (locationContext.startsWith('BELvue')) {
```

Comment:
@oliviervd, verliezen we hiermee niet een hoop informatie of wat is het doel van deze change?